#!/usr/bin/env python
PACKAGE = "ros_cutball"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("yellow_H1",    int_t,    0, "An Integer parameter", 0, 0, 180)
gen.add("yellow_S1",    int_t,    0, "An Integer parameter", 223, 0, 255)
gen.add("yellow_V1",    int_t,    0, "An Integer parameter", 0, 0, 255)
gen.add("yellow_H2",    int_t,    0, "An Integer parameter", 145, 0, 180)
gen.add("yellow_S2",    int_t,    0, "An Integer parameter", 255, 0, 255)
gen.add("yellow_V2",    int_t,    0, "An Integer parameter", 255, 0, 255)


exit(gen.generate(PACKAGE, "ros_cutball", "yellow_hsv_"))